modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/messageprocessor/http/HttpBulkMessageProcessor.java

```diff
@@ -57,27 +52,20 @@ private boolean _flush() {
 		catch (IOException e)
 		{
 			logger.error(e.getMessage());
-			return false;
+			retval = false;
 		}
 		if (! jestResult.isSucceeded()) {
 			logger.error(jestResult.getErrorMessage());
-			return false;
+			retval = false;
 		}
 		bulk = new Bulk.Builder();
```

Comment:
I think, we could extend `try-catch` with a `finally`, where we could assign a new `Bulk.Builder` to `bulk`·
(and in this case we can keep early returns)

Reply:
Indeed, it resulted in a much cleaner code. Thanks for the comment.